modules/renter/hostdb/scan_test.go

```diff
@@ -0,0 +1,156 @@
+package hostdb
+
+import (
+	"net"
+	"testing"
+	"time"
+
+	"github.com/NebulousLabs/Sia/encoding"
+	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/types"
+)
+
+// TestDecrementReliability tests the decrementReliability method.
+func TestDecrementReliability(t *testing.T) {
+	hdb := bareHostDB()
+
+	// Decrementing a non-existent host should be a no-op.
+	// NOTE: can't check any post-conditions here; only indication of correct
+	// behavior is that the test doesn't panic.
+	hdb.decrementReliability("foo", types.NewCurrency64(0))
```

Comment:
you should at least drop a TODO in the code to suggest that a `build.Critical` should be thrown at some point in the future.